gooddata.gemspec

```diff
@@ -55,8 +55,8 @@ Gem::Specification.new do |s|
   s.add_dependency 'docile', '~> 1.1'
   s.add_dependency 'erubis', '~> 2.7'
   s.add_dependency 'gli', '~> 2.15'
-  s.add_dependency 'gooddata_datawarehouse' if RUBY_PLATFORM == 'java'
-  s.add_dependency 'gooddata-dss-jdbc', '0.1.12' if RUBY_PLATFORM == 'java'
+  s.add_dependency 'gooddata_datawarehouse', '~> 0.0.8' if RUBY_PLATFORM == 'java'
+  s.add_dependency 'gooddata-dss-jdbc', '~> 0.2.4' if RUBY_PLATFORM == 'java'
```

Comment:
warning: Gemspec/OrderedDependencies: Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `gooddata-dss-jdbc` should appear before `gooddata_datawarehouse`. (rubocop)